plugins/modules/nd_backup.py

```diff
@@ -28,38 +30,87 @@
   encryption_key:
     description:
     - The encryption_key for a backup file.
+    - A minimum of 8 alphanumeric characters is required.
     type: str
   file_location:
     description:
     - The download path and file name for a backup.
+    - When O(file_location) is specified, the backup will be created and automatically downloaded to the local machine at the designated path.
+    aliases: [ local_path, path ]
     type: str
   backup_key:
     description:
     - The key generated by ND during creation of a backup.
     - This key is required when querying or deleting a backup among multiple backups that have the same name.
     - This key can be obtained by querying the backup.
+    - This parameter is not supported on ND v3.2.1 and later.
+    type: str
+  backup_type:
+    description:
+    - This parameter is only supported on ND v3.2.1 and later.
+    - The O(backup_type=config_only) option creates a snapshot that specifically captures the configuration settings of the Nexus Dashboard.
+    - The O(backup_type=full) option creates a complete snapshot of the entire Nexus Dashboard.
+    - When unspecified, the parameter defaults to O(backup_type=config_only).
```

Comment:
```suggestion
    - The O(backup_type=config_only) option creates a snapshot that specifically captures the configuration settings of the Nexus Dashboard.
    - The O(backup_type=full) option creates a complete snapshot of the entire Nexus Dashboard.
    - When unspecified, the parameter defaults to O(backup_type=config_only).
    - This parameter is only supported on ND v3.2.1 and later.
```
I think the supported version line can be at the end of the description.